### DISCUSSION TITLE
Sorting feature

### DISCUSSION THREAD

**Discussion by azzamsa** [firefly-iii/firefly-iii#4393] [General]
Hi.

Can we do sorting in `accounts/asset`, `accounts/expense`, etc ?
Such clicking in `name` will sort alphabetically, and clicking in `current balance` will sort by amount.

![image](https://user-images.githubusercontent.com/17734314/107896143-09679f80-6f68-11eb-98ba-2072745bfc83.png)

```
version: https://demo.firefly-iii.org/accounts/asset
```

**Comment by JC5** [comment]
Yes, I plan to add something like that in the new layout.
